.github/workflows/pyre.yml

```diff
@@ -14,31 +14,20 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v1
+      - uses: actions/setup-python@v2
 
-      - name: Install Dependencies
+      - name: Install Dependencies and pyre-check
```

Comment:
Why the change here? I think of this as "dependencies for the action" and Pyre is just a normal dependency.

Reply:
I wanted to give it some special status but yeah it's fine either way :)